addons/mail/static/src/js/chat_manager.js

```diff
@@ -506,6 +507,8 @@ function on_toggle_star_notification (data) {
             chat_manager.bus.trigger('update_message', message);
         }
     });
+    starred_counter = _.where(messages, {is_starred: true}).length;
+    chat_manager.bus.trigger('update_starred', starred_counter);
```

Comment:
@ged-odoo ,
Instead of increment/decrement we counted the number of starred messages because in slower connection it will generate wrong value(If user click multiple times quickly on star button)